dependency.lic

```diff
@@ -329,6 +329,11 @@ class ScriptManager
     @versions = nil
     repo_scripts.each { |(name, _)| get_script(name, force) }
     autostarts.each do |script|
+      if script == 'drinfomon'
+        respond "\n---\ndrinfomon found in autostarts. Attempting to remove it now. If unsuccessful, it can be safely removed with `;e stop_autostart('drinfomon')`\n---\n"
```

Comment:
Should `;e` be `#{$clean_lich_char}e`?

Reply:
I've changed it, but just to note that genie, the only FE I know of that uses a different character, now also accepts `;`